enocean/__init__.py

```diff
@@ -669,7 +696,67 @@ def send_learn_switch(self, id_offset=0):
             return
         self.logger.info("enocean: sending learn telegram for switch command")
         self._send_radio_packet(id_offset, 0xA5, [0x01, 0x00, 0x00, 0x00])
-
+        
```

Comment:
The content of the functions below are fine but please combine those into one single on/off function, i.e. see send_switch function prototype. Additionally, please not not use device specific names, like nodon. That way we would have thousands of different functions for Eltako, Nodon, Gira etc. Use the enocean way instead and name them by EEP. Thanks!